templates/basic/server.js

```diff
@@ -0,0 +1,11 @@
+const express = require('express')
```

Comment:
Run `Format Document` before committing to have better readability?

Reply:
sure